src/libraries/System.Private.CoreLib/src/System/Threading/Tasks/Task.cs

```diff
@@ -5480,7 +5480,8 @@ public static Task<TResult> Run<TResult>(Func<Task<TResult>?> function, Cancella
         /// The <paramref name="delay"/> is less than -1 or greater than the maximum allowed timer duration.
         /// </exception>
         /// <exception cref="System.ObjectDisposedException">
-        /// The provided <paramref name="cancellationToken"/> has already been disposed.
+        /// The associated <see cref="System.Threading.CancellationTokenSource">CancellationTokenSource</see>
+        /// of the provided <paramref name="cancellationToken"/> has already been disposed.
```

Comment:
💡 If there is already a standard message for the scenario, we should probably use that. Otherwise, I would suggest this alternative:

```suggestion
        /// The <see cref="CancellationTokenSource"/> associated
        /// with <paramref name="cancellationToken"/> has already been disposed.
```

Reply:
I adapted what I wrote from `CancellationToken.cs`:

```
/// <exception cref="System.ObjectDisposedException">The associated <see
/// cref="System.Threading.CancellationTokenSource">CancellationTokenSource</see> has been disposed.</exception>
```

I don't know if there are similar messages in other places.

Reply:
I agree that your suggestion is better.